lib/dataset/test/bin_test.cpp

```diff
@@ -819,3 +819,75 @@ TEST(BinTest, bin_by_noncontiguous_int_group) {
 
   EXPECT_EQ(bin(table, {}, {non_cont_group}), bin(table, {}, {cont_group}));
 }
+
+TEST(BinTest, nan_values_are_dropped) {
```

Comment:
```suggestion
TEST(BinTest, points_with_nan_coord_values_are_dropped) {
```
I suppose? To differentiation from, say `nanhist`.

Same for other test names.